src/main/java/org/folio/circulation/rules/Drools.java

```diff
@@ -153,6 +153,44 @@ public String noticePolicy(MultiMap params, Location location) {
     return match.noticePolicyId;
   }
 
+  /**
+   * Calculate the overdue fine  policy for itemTypeName and requestTypeName.
```

Comment:
Can you remove the duplicate space in

    fine  policy

Reply:
Fixed.